tests/unit/src/Internal/LoggerTest.php

```diff
@@ -0,0 +1,300 @@
+<?php
+/**
+ * Class LoggerTest
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Tests;
+
+use WCPAY_UnitTestCase;
+use WC_Logger;
+use WC_Log_Levels;
+use WC_Payment_Gateway_WCPay;
+use Exception;
+use WCPay\Core\Mode;
+use WCPay\Internal\Logger;
+
+/**
+ * Internal Logger tests.
+ *
```

Comment:
RadoCS: Let's remove this empty line :)

Reply:
I think its done, let me know if you find anymore blank lines :)